src/java/org/apache/cassandra/db/tries/Direction.java

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright DataStax, Inc.
```

Comment:
```
08:28:51  /home/jenkins/workspace/ds-cassandra-pr-gate_PR-1079/.build/build-rat.xml:89: Some files have missing or incorrect license information. Check RAT report in /home/jenkins/workspace/ds-cassandra-pr-gate_PR-1079/build/rat.txt for more details! \n Unapproved licenses:
08:28:51  
08:28:51    /home/jenkins/workspace/ds-cassandra-pr-gate_PR-1079/src/java/org/apache/cassandra/db/tries/Direction.java
```

Looks like CI doesn't like this license.

Reply:
Done